Documentation/object-store-crd.md

```diff
@@ -81,6 +82,7 @@ The gateway settings correspond to the RGW daemon settings.
 
 - `type`: `S3` is supported
 - `sslCertificateRef`: If the certificate is not specified, SSL will not be configured. If specified, this is the name of the Kubernetes secret that contains the SSL certificate to be used for secure connections to the object store. Rook will look in the secret provided at the `cert` key name. The value of the `cert` key must be in the format expected by the [RGW service](http://docs.ceph.com/docs/master/install/install-ceph-gateway/#using-ssl-with-civetweb): "The server key, server certificate, and any other CA or intermediate certificates be supplied in one file. Each of these items must be in pem form."
+- `host`: The DNS name for the object store. The name must be resolvable by DNS in your cluster. The name must also be provided as the host name in the URL of s3 client requests. The default is: `<rgwService>-<storeName>.<namespace>`. In the example above it would be: `rook-ceph-rgw-my-store.rook`.
```

Comment:
Why does this have to be configurable? In K8S you can already access a service endpoint via DNS. Can we not use that instead?
And if we need it, the `host` term seems very generic and overloaded. Can we change it to `domainName` instead?